README.md

```diff
@@ -67,6 +68,31 @@ General Assemblyline documentation can be found at: https://cybercentrecanada.gi
 
 Service Assemblyline basé sur les signatures et axé sur la généalogie des fichiers.
 
+## Détails du service
+
+### Signatures
+
+(BETA) Les signatures sont définies en tant que configuration de service et chaque signature suit le format suivant:
+
+```yaml
+config:
+  signatures:
+    exe_from_office_document: # Nom de la signature
+      pattern: « document/office/.+,ROOT\|executable/windows/(?:pe|dll)(?:32|64),EXTRACTED » # Motif Regex
```

Comment:
As technical documentation, this should still use English bracket " instead of French bracket «»